src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -302,6 +307,27 @@ export class HuiAreaCard extends LitElement implements LovelaceCard {
     `;
   }
 
+  private _renderMediaBadge(): TemplateResult<1> | typeof nothing {
+    const states = this._computeActiveMediaStates();
+
+    if (states.length === 0) {
+      return nothing;
+    }
+
+    const firstState = states[0];
```

Comment:
Why do we use the first state? Shouldn't `playing` take president over `paused` for example?